src/scaler/spreadscaler/mod.rs

```diff
@@ -58,6 +58,7 @@ impl<S: ReadStore + Send + Sync + Clone> Scaler for ActorSpreadScaler<S> {
     }
 
     async fn status(&self) -> StatusInfo {
+        let _ = self.reconcile().await;
```

Comment:
My concern with doing it this way is that a reconcile just to get the status could theoretically cause a backoff period to occur:

1. Reconcile triggers
2. An event we are waiting for doesn't arrive or is missed
3. We wait the full 30s for actor scaler to backoff

I don't think this is blocking but I have an unproven worry this could cause some cases where wadm would appear "slow" to reconcile

Reply:
I think your specific concern is alleviated since this is inside of the Spreadscaler itself, we are not causing a backoff because it's not a Backoff aware spreadscaler. I do agree with you though that we have to be pretty careful about reconciling the generic Scaler trait.